src/Game.js

```diff
@@ -0,0 +1,41 @@
+export default class Game {
+    constructor() {
+        // this.alive = true;
+        this.player1;
+        this.player2;
+        this.wordToGuess = '';
+        this.attemptedGuesses = []; // All of these will be displayed as guesses
+        this.maxWrongAttempts = 6;
+        this.wrongGuesses = 0;
+        this.correctGuesses = []; // These will be displayed as correct guesses only
+    }
+    reviewAttempt(guess) {
+        this.attemptedGuesses.push(guess);
+        if (!this.wordToGuess == guess || !this.wordToGuess.includes(guess)) {
+            this.wrongGuesses += 1;
+            this.wrongAttempt();
+        } else {
+            this.correctGuesses.push(guess);
+            this.correctAttempt();
+        }
+    }
+    wrongAttempt() {
+        if (this.wrongGuesses >= this.maxWrongAttempts) {
+            alert('The man is dead');
+            location.reload();
+        }
+    }
+    correctAttempt() {
+        let theWordSplit = this.wordToGuess.split('');
+        if (this.correctGuesses(-1) == this.wordToGuess) {
+            winGame();
+        }
+        if (this.correctGuesses.includes(theWordSplit)) {
```

Comment:
`this.correctGuesses` is an array of strings.
`theWordSplit` is an array of strings.

Logically, this can never resolve to `true`. Maybe try `this.correctGuesses.includes(theWordSplit.join(''))`

Reply:
`this.correctGuesses.slice[-1]` is the most recent correct guess, `this.correctGuesses` can contain guesses of letters or words. 

Line 30 is checking to see if the most recent correct guess equals `this.wordToGuess` (which itself is equal to theWordSplit.join('')).

Reply:
I'm not talking about line 30, I'm talking about line 33